extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -160,16 +160,6 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
   }
 
-  /**
-   * Prefetches and preconnects URLs related to the analytics.
-   * @param {boolean=} opt_onLayout
-   * @override
-   */
-  preconnectCallback(opt_onLayout) {
-    const url = getIframeTransportScriptUrl(this.getAmpDoc().win);
-    this.preconnect.preload(url, 'script');
```

Comment:
Moved to lines 334-335 below. That is the point of the PR: don't do this until we know that it'll be needed.